clients/python_clients/cloudmqtt-3dprinters.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: iso-8859-15 -*-
```

Comment:
Just wondering, the filename says 3d printer but the contents don't seem 3d printer related? Is this perhaps the wrong file?

Reply:
Oh just realized this PR goes from master > develop